### PR TITLE
Update to 0.12.8, refresh recipe, noarch

### PR DESCRIPTION
**Pull request by dbast**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dbast** [comment]
Ready for review :)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,50 +1,50 @@
...
 source:
-    fn: {{ name }}-{{ version }}.tar.gz
-    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    {{ hash_type }}: {{ hash_val }}
+  url: "https://github.com/jupyter-incubator/sparkmagic/archive/{{ version }}.tar.gz"
+  sha256: "1f69766c86b4501b47abe67494a5f5001a466c7d6d5f56cf664e776bbe8c2dae"
 
 build:
-    number: 0
-    skip: true  # [win]
```
Comment (ericdill) [inline recipe/meta.yaml]: what changed to make this recipe compatible with windows?

**Comment by ericdill** [comment]
going to assume that noarch python is fine

**Comment by ericdill** [comment]
Thanks for the PR @dbast

**Pull request merged**